controllers/upgrade.go

```diff
@@ -97,6 +108,11 @@ func (r *RollingUpgradeReconciler) ReplaceNodeBatch(rollingUpgrade *v1alpha1.Rol
 			// Is drained?
 
 			// Terminate - set lastTerminateTime
+			if err := r.Auth.TerminateInstance(target); err != nil {
+				r.Info("Instance termination failed. Will retry in next reconcile", "name", rollingUpgrade.NamespacedName(), "instance", aws.StringValue(target.InstanceId))
```

Comment:
nit: probably better to log the error, suggest you also simplify the message e.g.
```golang
r.Info("failed to terminate instance", "name", rollingUpgrade.NamespacedName(), "instance", aws.StringValue(target.InstanceId), "error", err)
```
in general I think also more appropriate to keep all log messages lower-case, but that's more personal preference.

Reply:
I first tried logging the error. The entire console becomes too messy with the long error messages. But the rollup succeeds anyway. I thought these long error messages could mislead the observer. WDYT?

Regarding the case, let's keep it uniform. I will keep all the logging in lowercase.

Reply:
Well, we need to know why the call failed.. I think the error should not be too long, can you give an example to how the log was messy? did it add a stack trace?

Reply:
Yup. It adds stacktrace.

Reply:
I have added the error log messages for now. You are right, we should know why it failed.
I think we can do a final review of all the logging that goes to the console as a ticket.